src/otx/core/model/entity/instance_segmentation.py

```diff
@@ -204,7 +241,7 @@ def _customize_outputs(
             )
             scores.append(torch.tensor([output.score for output in output_objects]))
             masks.append(torch.tensor([output.mask for output in output_objects]))
-            labels.append(torch.tensor([output.id for output in output_objects]))
+            labels.append(torch.tensor([output.id - 1 for output in output_objects]))
```

Comment:
If i-seg model is converted to OV, empty label should be added as first element. So, it should be aligned in this part also.